railties/test/application/configuration_test.rb

```diff
@@ -928,12 +928,13 @@ def index
       assert_equal "3b7cd727ee24e8444053437c36cc66c3", app.secret_key_base
     end
 
-    test "config.secret_key_base over-writes a blank app.secret_key_base" do
+    test "app.secret_key_base falls back to config.secret_key_base in production" do
```

Comment:
I repurposed this test because it was doing the exact same thing as the test above